test.hxml

```diff
@@ -11,4 +11,4 @@
 --remap flash:openfl
 
 ## Neko
--cmd neko build/neko_test.n
\ No newline at end of file
+-neko build/neko_test.n
```

Comment:
You launch `haxe test.hxml`, it find inside the `-neko build/neko_test.n ` which indicates it to build a neko file from the code, then it finds the `-cmd` line which tells it to run the command like in a shell. So you wouldn't execute the test manually after that.